observability/logging/cluster-logging-support.adoc

```diff
@@ -51,6 +51,11 @@ LokiStack development is ongoing. Not all APIs are currently supported.
 include::modules/cluster-logging-maintenance-support-list.adoc[leveloffset=+1]
 include::modules/unmanaged-operators.adoc[leveloffset=+1]
 
+[id="support-exception-for-coo-logging-ui-plugin"]
+== Support exception for the Logging UI Plugin
```

Comment:
Is it "Logging UI Plugin" or "logging UI plugin"? I see a mix throughout.

Reply:
This was raised during the peer review. Here is my reply: https://github.com/openshift/openshift-docs/pull/83306#discussion_r1796646444.